src/python/pants/backend/python/buildozer/buildozer.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import subprocess
```

Comment:
this should use `from pants.util.process_handler import subprocess` instead of the stdlib variant